safemysql.class.php

```diff
@@ -616,6 +620,20 @@ private function escapeInt($value)
 		return $value;
 	}
 
+	private function escapeBool($value)
+	{
+		if ($value === true || $value === 1 || $value === '1')
```

Comment:
BOOL columns get retrieved from MySQL as numeric strings, so it seems sensible to let them be put back in in the same way.